pkg/expression/bench_test.go

```diff
@@ -1300,7 +1300,7 @@ func genVecExprBenchCase(ctx sessionctx.Context, funcName string, testCase vecEx
 // testVectorizedEvalOneVec is used to verify that the vectorized
 // expression is evaluated correctly during projection
 func testVectorizedEvalOneVec(t *testing.T, vecExprCases vecExprBenchCases) {
-	ctx := mock.NewContext()
+	ctx := createContext(t)
```

Comment:
We use `createContext(t)` instead of `mock.NewContext()` to force to append to warnings for truncate error